### ISSUE TITLE
please publish to maven central

### ISSUE THREAD

**Issue by jgangemi** [Ullink/simple-slack-api#125]
please consider publishing this library to maven central, otherwise it requires me to define this section in my pom

```xml
  <repositories>
    <repository>
      <id>central</id>
      <url>http://repo1.maven.org/maven2</url>
      <snapshots>
        <enabled>false</enabled>
      </snapshots>
    </repository>
    <repository>
      <id>oss-snapshots-repo</id>
      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
      <releases>
        <enabled>false</enabled>
      </releases>
      <snapshots>
        <enabled>true</enabled>
      </snapshots>
    </repository>
    <repository>
      <id>jitpack.io</id>
      <url>https://jitpack.io</url>
    </repository>
  </repositories>
```
which http://central.sonatype.org/pages/requirements.html#a-complete-example-pom recommends against.

**Comment by Stephan202** [comment]
Older versions [were](https://repo1.maven.org/maven2/com/ullink/slack/simpleslackapi/) pushed to Maven Central. This practice was abandoned?

**Comment by jgangemi** [comment]
odd - when i searched last night i didn't see `0.6.0` in the repo. it was also around midnight, so that could explain a lot...

going to leave this open just to get a final confirmation on if releases will continue to be published there.

**Comment by bcorne** [comment]
No, it was not abandonned, but I think it is due to the fact I changed the key I'm using to sign the jars. I will check that.

**Comment by jgangemi** [comment]
i think what was throwing me is i search for `simple-slack-api` which yields this:
http://search.maven.org/#search%7Cga%7C1%7Csimple-slack-api

instead of searching for `simpleslackapi` which yields this:
http://search.maven.org/#search%7Cga%7C1%7Csimpleslackapi

and has the latest version. sorry for any confusion - can this request yield an update to the `README` that states the coordinates of where the jar is available?

**Comment by bcorne** [comment]
sure it can, I'll do that

**Comment by yegor256** [comment]
@bcorne the latest version is in Maven Central now. I guess, this issue should be closed.
